app/_src/gateway/migrate-to-new-dns-client.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Migrate to the new DNS client
+---
+This guide walks you through migrating from the old DNS client to the new one. The new DNS client introduces a new standardized way to configure a service, and helps improve performance.
+
+## Migrate the DNS client configuration
+
+### Record types
+
+The new DNS client introduces some changes in the support for different record types. To avoid issues, make sure to check your configuration before migrating.
```

Comment:
```suggestion
The new DNS client introduces some changes in the support for different record types. To avoid issues, make sure to check your configuration before migrating.
```

Is there anything specific that I should be checking for? Record types?

Reply:
Yes, mainly the domain name for SRV records

Reply:
Write that specifically here.